src/styles/_catalog.scss

```
@@ -1,12 +1,31 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(auto-fill, minmax(200px 1fr));
+  gap: 48px;
```

add margin: 0 auto to center main container inside the body

```suggestion
  gap: 48px;
  margin: 0 auto;
```